### DISCUSSION TITLE
MARIADB error on delete

### DISCUSSION THREAD

**Discussion by Kry9toN** [Budibase/budibase#13221] [Help] [answered]
Anyone can help me for this issue?

![image](https://github.com/Budibase/budibase/assets/44697929/793290f4-e29a-433d-885f-e3e8acac2401)

**Comment by mike12345567** [comment, marked as answer]
Hi @Kry9toN - if you're self hosting please update to the latest version of Budibase, it will solve this issue.

**Reply by Kry9toN** [reply]
thanks is works
